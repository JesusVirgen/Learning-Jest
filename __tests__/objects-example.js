const customer = {
    name: 'Jesus Virgen',
    balance: 500,
    type: 'Premium'
};

describe('Customer testing', () => {
    test('When customer is premium', () => {
        expect( customer.balance ).toBeGreaterThan(400);
    });
    test('When customer is Jesus Virgen', () => {
        expect(customer).toMatchSnapshot();
    });
    test('Is not another customer', () => {
        expect(customer.name).not.toBe('Jose');
    });
    test('When customer is not premium', () => {
        expect( customer.balance ).not.toBe(400);
    });
});