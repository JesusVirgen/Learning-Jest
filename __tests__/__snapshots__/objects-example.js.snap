// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer testing When customer is Jesus Virgen 1`] = `
{
  "balance": 500,
  "name": "Jesus Virgen",
  "type": "Premium",
}
`;
